docgen/content-sources/toc.yaml

```diff
@@ -15,11 +15,19 @@ toc:
       - title: 'ChangeJson'
         path: /docs/reference/functions/cloud_functions_.changejson.html
 
+  - title: 'functions.HandlerBuilder'
```

Comment:
the title should be `functions.handler`, since that's how a developer actually writes it in code. 
e.g.
```exports.myFunction = functions.handler.database.ref.onWrite(...)```

Reply:
This section can also be lower in the list (it can be the last item actually, since regular CF3 developers won't need it, and extensions docs will link directly to the handlers page.)